src/main/java/com/feedzai/commons/sql/abstraction/engine/configuration/PdbProperties.java

```diff
@@ -150,6 +152,14 @@
      * space in the DB server. This depends on the database implementation.
      */
     public static final String DISABLE_LOB_CACHING = "pdb.disable_lob_caching";
+    /**
+     * Property that indicates the time, in seconds, of login timeout in the socket.
```

Comment:
I'm OK with keeping the property name as is, but I think there's missing some more context on the javadoc. For instance, something like: "... in seconds, to wait for the database connection to be established" is more easy to understand than "login timeout in the socket".

Reply:
Already fixed. Thanks for the comment.